.gitattributes

```diff
@@ -0,0 +1,32 @@
+# All text files normalized to lf in repo (overrides local core.autocrlf)
+* text=auto
```

Comment:
Does this mean that any file that isn't listed as binary is treated as text?  It seems safter to do it the other way.